builders/invoke-workflow.psm1

```diff
@@ -0,0 +1,52 @@
+function Invoke-Workflow {
+    param (
+        [string] $Version,
+        [bool] $PublishRelease
+    )
+    
+    $payload = @{
+        "ref" = "main"
+        "inputs" = @{
+            "VERSION" = "$Version"
+            "PUBLISH_RELEASES" = "$PublishRelease"
+        }
+    } | ConvertTo-Json
+    $headers = @{
+        Authorization="Bearer $env:PERSONAL_TOKEN"
+    }
+
+    Invoke-RestMethod -uri "$env:GITHUB_API_URL/repos/$env:GITHUB_REPOSITORY/actions/workflows/python-builder.yml/dispatches" -method POST -headers $headers -body $payload
```

Comment:
What do you think to create a separate variable for uri part?

`$env:GITHUB_API_URL/repos/$env:GITHUB_REPOSITORY/actions`

Reply:
Move to separate variable